cmd/supernode/app/root.go

```diff
@@ -21,24 +21,28 @@ import (
 	"os"
 	"path"
 	"reflect"
+	"time"
 
 	"github.com/dragonflyoss/Dragonfly/pkg/dflog"
 	"github.com/dragonflyoss/Dragonfly/pkg/errortypes"
 	"github.com/dragonflyoss/Dragonfly/pkg/fileutils"
 	"github.com/dragonflyoss/Dragonfly/pkg/netutils"
+	"github.com/dragonflyoss/Dragonfly/pkg/rate"
 	"github.com/dragonflyoss/Dragonfly/pkg/stringutils"
 	"github.com/dragonflyoss/Dragonfly/supernode/config"
 	"github.com/dragonflyoss/Dragonfly/supernode/daemon"
 
+	"github.com/mitchellh/mapstructure"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"gopkg.in/yaml.v2"
 )
 
 var (
-	configFilePath = config.DefaultSupernodeConfigFilePath
-	cfg            = config.NewConfig()
-	options        = NewOptions()
+	defaultConfigFilePath = config.DefaultSupernodeConfigFilePath
+	defaultBaseProperties = config.NewBaseProperties()
```

Comment:
I think we can move these default value into `setupFlags` since it's not used anywhere else.